ARCHITECTURE.md

```diff
@@ -5,7 +5,7 @@ find it useful if you want to contribute to Cargo or if you are
 interested in the inner workings of Cargo.
 
 The purpose of Cargo is to formalize a canonical Rust workflow, by automating
-the standard tasks associated with distributing software.  Cargo simplifies
+the standard tasks associated with distributing software. Cargo simplifies
```

Comment:
This isn't _strictly_ (or objectively) wrong:

https://en.wikipedia.org/wiki/Sentence_spacing

Reply:
It's not, but it's definitely considered old-fashioned, and *usually* prescribed against these days. The issue is, when you don't have actual typesetting (just a monospace font), it becomes more up-for-debate.

Reply:
I'm fine with dropping it.  But I use it because it's useful in vim.

Reply:
Okay, thanks. And fair enough.